app/assets/javascripts/usa_map.js

```diff
@@ -28,20 +28,19 @@ $(document).ready(function() {
         })
         .attr("d", path)
     });
-
-    /* The below code will be coordinates obtained from using the geocoder gem to get
-    long/lat of cities The code is : Geocoder.search(city)[0].data["geometry"]["location"]["lng"]*/
-
-    var marks = [{long: -77, lat: 43},{long: -100, lat: 44},{long: -70, lat: 43}];
+    var marks = [];
+    $(".map").data("coordinates").forEach(function(coordinate) {
+      marks.push(coordinate);
+    })
 
     svg.selectAll(".mark")
         .data(marks)
         .enter()
         .append("image")
-        .attr('class','mark')
+        .attr('class', function(d,i) { return "mark" + (i) })
```

Comment:
this gives each marker a unique class so I can select a marker in order to move them at some later point.